atomone/chain.json

```diff
@@ -0,0 +1,141 @@
+    },
+    "versions": [
+      {
+        "name": "v1.0.0",
+        "tag": "v1.0.0",
+        "recommended_version": "v1.0.0",
+        "compatible_versions": [
+          "v1.0.0"
+        ],
+        "cosmos_sdk_version": "v0.47.13",
+        "consensus": {
+          "type": "cometbft",
+          "version": "v0.37.5"
+        },
+        "height": 1,
+        "binaries": {
+          "linux/amd64": "https://github.com/atomone-hub/atomone/releases/download/v1.0.0/atomoned-v1.0.0-linux-amd64",
+          "linux/arm64": "https://github.com/atomone-hub/atomone/releases/download/v1.0.0/atomoned-v1.0.0-linux-arm64",
+          "darwin/amd64": "https://github.com/atomone-hub/atomone/releases/download/v1.0.0/atomoned-v1.0.0-darwin-amd64",
+          "darwin/arm64": "https://github.com/atomone-hub/atomone/releases/download/v1.0.0/atomoned-v1.0.0-darwin-arm64",
+          "windows/amd64": "https://github.com/atomone-hub/atomone/releases/download/v1.0.0/atomoned-v1.0.0-windows-amd64.exe",
+          "windows/arm64": "https://github.com/atomone-hub/atomone/releases/download/v1.0.0/atomoned-v1.0.0-windows-arm64.exe"
+        },
+        "sdk": {
+          "type": "cosmos",
+          "version": "v0.47.13"
+        }
+      }
+    ]
+  },
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/atomone/images/atomone.png",
+    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/atomone/images/atomone.svg"
+  },
+  "description": "The Cosmos community, at a crossroads, confronts divergent views on key aspects such as mission, tokenomics, and security philosophy. AtomOne emerges as a beacon, offering an alternative fork to navigate these waters, equipped to handle contingencies and embodying a bastion for diverse political thought.",
+  "peers": {
+    "seeds": [],
+    "persistent_peers": [
+      {
+        "id": "9dac79c27e4dba77cb22b0b25933fee6c8121cf7",
+        "address": "72.46.84.243:26656"
+      }
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "https://atomone-rpc.allinbits.com:443",
+        "provider": "AllInBits"
+      }
+    ],
+    "rest": [
+      {
+        "address": "https://atomone-api.allinbits.com",
+        "provider": "AllInBits"
+      }
+    ],
+    "grpc": [
+      {
+        "address": "https://atomone-grpc.allinbits.com:443",
+        "provider": "AllInBits"
+      }
+    ]
+  },
+  "explorers": [
+    {
+      "kind": "ping.pub",
+      "url": "https://explorer.allinbits.com/atomone",
+      "tx_page": "https://explorer.allinbits.com/atomone/tx/${txHash}",
+      "account_page": "https://explorer.allinbits.com/atomone/account/${accountAddress}"
+    }
+  ],
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/atomone/images/atomone.png",
+      "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/atomone/images/atomone.svg",
+      "theme": {
+        "primary_color_hex": "#65d6d4"
```

Comment:
How was this color selected?

Reply:
To be honest, I tried to find out where this color is used in cosmos.directory and couldn't find any place where it was used. So I concluded that it didn't matter and left it unchanged.

Reply:
Ok. Well if you aren't going to select the correct color, then please omit the property

Reply:
Done here 9f0c24e